src/bosh_azure_cpi/lib/cloud/azure/blob_manager.rb

```diff
@@ -5,15 +5,14 @@ class BlobManager
     include Helpers
 
     MAX_CHUNK_SIZE = 2 * 1024 * 1024 # 2MB
-    HASH_OF_2MB_EMPTY_CONTENT = 'b2d1236c286a3c0704224fe4105eca49' # The hash value of 2MB empty content
     TIMEOUT_FOR_BLOB_OPERATIONS = 120 # Timeout in seconds for Blob Service Operations. https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations
 
     def initialize(azure_properties, azure_client2)
       @parallel_upload_thread_num = 16
       @parallel_upload_thread_num = azure_properties['parallel_upload_thread_num'].to_i unless azure_properties['parallel_upload_thread_num'].nil?
       @azure_properties = azure_properties
       @azure_client2 = azure_client2
-
+      @two_mb_empty_content = Array.new(MAX_CHUNK_SIZE, 0)
```

Comment:
It seems no need to mention the size in the name. The size depends on `MAX_CHUNK_SIZE`. So, how about removing `two_mb`?

Reply:
good suggestion.